containers/python-redpanda/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.4"
+
+services:
+  deephaven:
+    image: ghcr.io/deephaven/server:${VERSION:-latest}
+    ports:
+      - "${DEEPHAVEN_PORT:-10000}:10000"
+    volumes:
+      - ./data:/data
+    environment:
+      - START_OPTS=-Xmx4g
+
+  redpanda:
+    command:
+    - redpanda
+    - start
+    - --smp
+    - '1'
+    - --reserve-memory
+    - 0M
+    - --overprovisioned
+    - --node-id
+    - '0'
+    - --kafka-addr
+    - PLAINTEXT://0.0.0.0:29092,OUTSIDE://0.0.0.0:9092
+    - --advertise-kafka-addr
+    - PLAINTEXT://redpanda:29092,OUTSIDE://localhost:9092
```

Comment:
I think we should adopt the formatting style redpanda uses for this:

```yaml
    command:
      - redpanda
      - start
      - --kafka-addr internal://0.0.0.0:9092,external://0.0.0.0:19092
      - --advertise-kafka-addr internal://redpanda:9092,external://localhost:19092
      - --schema-registry-addr internal://0.0.0.0:8081,external://0.0.0.0:18081
      - --smp 1
      - --memory 1G
      - --mode dev-container
```

I've modified this from https://docs.redpanda.com/current/get-started/quick-start/; we should test it in all of the different compose files to make sure it works. Do we use the schema registry for any of our examples?

Reply:
I don't think we do. I've made updates that make our syntax consistent with Redpanda and tested it to make sure it works with our Kafka documentation.

Reply:
I don't think we're consistent w/ the redpanda quickstart - they use `internal` and `external`, and use `dev-container`.